source/less/blocks/achievement.less

```
@@ -0,0 +1,12 @@
+.achievement {
+  .achievement__text {
```

```suggestion
  &__text {
```
так же нужно делать)
это нужно поправить во всех файлах